src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,13 +1,44 @@
 package wolox.training.controllers;
 
-import org.springframework.stereotype.Controller;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
-@Controller
+@RestController
+@RequestMapping("/api/books")
 public class BookController {
 
+    @Autowired
+    private BookRepository bookRepository;
+
```

Comment:
A good practice mentioned at this point in the training is to add java docs in each of the controllers. They are also used to describe and document methods that may be considered complex.